client/controller/controller.js

```diff
@@ -0,0 +1,355 @@
+
+app.controller('AppCtrl',function($scope, $http, filterdata) {
+
+
+    // ----> Code from autocomplete.js <------
+    var inputFrom = document.getElementById('input');
```

Comment:
Use `$scope` or angularjs built-ins instead of `document.getElementById` in angular js